src/chat/chat_server.py

```diff
@@ -0,0 +1,20 @@
+def create_connection_pool():
+	pass
+
+def store_in_DB():
```

Comment:
Following snake case `store_in_db`, this would maintain uniformity. Same for `fetch_from_db`